memcached.c

```diff
@@ -6343,6 +6359,10 @@ static void usage(void) {
            "-A, --enable-shutdown     enable ascii \"shutdown\" command\n"
            "-a, --unix-mask=<mask>    access mask for UNIX socket, in octal (default: 0700)\n"
            "-l, --listen=<addr>       interface to listen on (default: INADDR_ANY)\n"
+#ifdef TLS
+           "                          if TLS/SSL is enabled, 'notls' prefix can be used to exclude ports\n"
```

Comment:
"can be used to disable for specific listeners". doesn't have anything to do with ports anymore :)

Reply:
+1